### PR TITLE
Initialize Creation Of Game

### PR DESCRIPTION
**Pull request by alexisljn**
| Q             | A
| ------------- | ---
| Is bugfix?    | Yes/No
| New feature?  | Yes/No
| Fixed issues  | Fixes # if any

Optional description...

### CODE REVIEW

**Comment by alexisljn** [comment]
Review Requested

File: frontend/src/components/pages/CreateGame/views/ChooseGameMode.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            const response = await Util.sendJsonToAPI('game/modes', {plan: userPlan});
+
+            if (!response.ok) throw `${response.status} : ${response.statusText}`;
+
+            const responseData = await response.json();
+
+            return responseData.gameModes;
+        } catch (error) {
+            throw error;
+        }
```
Comment (EmileCalixte) [inline frontend/src/components/pages/CreateGame/views/ChooseGameMode.js:50]: Bad usage of try/catch : this catch is useless

**Review by EmileCalixte — CHANGES_REQUESTED** [review]

**Review by aekix — APPROVED** [review]

File: backend/controllers/GameController.js [hunk 2]
```diff
@@ -25,19 +28,28 @@ class GameController extends MainController {
...
+                        allowed: true
+                    })
+                } else {
+                    allowedGameModes.push({
+                        classname: gameMode.CLASSNAME,
+                        label: gameMode.LABEL,
+                        description: gameMode.DESCRIPTION,
+                        allowed: false
+                    })
                 }
```
[the remark this answers is not in the document]
Comment (alexisljn) [reply backend/controllers/GameController.js:53]: You're right

File: frontend/src/components/App.js [hunk 3]
```diff
@@ -44,7 +44,7 @@ class App extends React.Component {
                                 <Route exact path="/" component={Home}/>
                                 <Route exact path="/login" render={() => <Login setUser={this.setUser} currentUser={this.state.user} />}/>
                                 <Route exact path="/register" component={Register}/>
-                                <Route exact path="/create-room/game-mode" component={CreateGame}/>
+                                <Route  path="/create-room/" component={CreateGame}/>
```
[the remark this answers is not in the document]
Comment (alexisljn) [reply frontend/src/components/App.js:47]: My bad brudda

File: frontend/src/components/pages/CreateGame/views/ChooseGameMode.js (hunk 1 above)
Comment (alexisljn) [reply frontend/src/components/pages/CreateGame/views/ChooseGameMode.js:50]: You didn't know this hack ? its called "twice catching".

No more jokes, I fix that asap

**Comment by alexisljn** [comment]
Review Requested

**Review by EmileCalixte — APPROVED** [review]
Great !

**Review by aekix — APPROVED** [review]

**Pull request merged**